python/desc/imsim/camera_readout.py

```diff
@@ -477,6 +483,8 @@ def write_fits_file(self, outfile, overwrite=True, run_number=None,
         output[0].header['RATEL'] = self.ratel
         output[0].header['DECTEL'] = self.dectel
         output[0].header['ROTANGLE'] = self.rotangle
+        # Write the seed used by the read noise and dark current.
+        output[0].header['RN_SEED'] = self.seed
```

Comment:
I wonder if we should somehow label all imSim specific headers with some sort of prefix?  IMSIM_ ?  But, I'm not sure of the length constraints etc.